src/main/resources/swagger.api/schemas/field.json

```diff
@@ -58,6 +58,10 @@
     "ecsOnly": {
       "description": "Indicates if the field is available only in ECS environments",
       "type": "boolean"
+    },
+    "isJsonbArray": {
+      "description": "Indicates whether to treat the array field as a JSONB array or a Postgres array for query generation and processing.",
```

Comment:
Can we add the default value to the description? It would potentially save some time later by keeping us from having to look that up constantly :)

Reply:
And really, explicitly setting a default in this JSON might be worth it, too, to avoid lots of null checks in FQM